apps/drupal/drupal.conf

```diff
@@ -130,93 +163,42 @@ location / {
         open_file_cache_min_uses 2;
         open_file_cache_errors off;
     }
+    location ~* \.(woff)$ {
+         add_header "Access-Control-Allow-Origin" "*";
+    }
 
     ## PDFs and powerpoint files handling.
-    location ~* ^.+\.(?:pdf|pptx?)$ {
-        expires 30d;
+    ##location ~* ^.+\.(?:pdf|pptx?)$ {
```

Comment:
@debojitcampd 

We cant comment this. On pages you can upload pdf brochure. Need to check if that works after these changes.